src/screens/ChatScreen/ChatScreen.tsx

```diff
@@ -60,10 +79,10 @@ export const ChatScreen: React.FC = observer(() => {
         onSendPress={handleSendPress}
         onStopPress={handleStopPress}
         user={user}
-        //onAttachmentPress={!context ? handlePickModel : undefined}
         isStopVisible={inferencing}
         textInputProps={{
-          editable: !!context,
+          editable: !!context && !inferencing,
```

Comment:
Got it. Made the change to allow users to allow edit. However disabled the send button while the model is inferencing so that it does not cause issues.